sdk/storage/storage-blob/src/generated/src/models/index.ts

```diff
@@ -4864,6 +5013,16 @@ export interface BlobCopyFromURLHeaders {
    * State of the copy operation identified by x-ms-copy-id. Possible values include: 'success'
    */
   copyStatus?: SyncCopyStatusType;
+  /**
+   * This response header is returned so that the client can check for the integrity of the copied
+   * content. This header is only returned if the source content MD5 was specified.
+   */
+  contentMD5?: Uint8Array;
+  /**
+   * This response header is returned so that the client can check for the integrity of the copied
+   * content.
+   */
+  xMsContentCrc64?: Uint8Array;
```

Comment:
>xMsContentCrc64 [](start = 2, length = 15)

is there a better client name than `xMsContentCrc64`? I remember seeing some discussion for other languages but couldn't find it.

Reply:
This comes directly from the swagger and is not changed in the latest one.
Do you have more context on the discussion as who are involved and how is it going?

Reply:
I can't find it. This is fine as I've seen other places where the same name is used.

Reply:
@ljian3377 At least should make sure the naming is consistency inside storage sdk for crc64.

Reply:
> @ljian3377 At least should make sure the naming is consistency inside storage sdk for crc64.

It should be consistent as most of the SDKs use the same swagger, right?